spec/support/graphql/response_parser.rb

```diff
@@ -1,11 +1,15 @@
 module GraphQL
   module ResponseParser
-    def execute_and_parse_graphql_response(query:, variables:)
+
+    attr_accessor :parse_graphql_response
+
+    def execute_and_parse_graphql_response(query:, variables:, current_user: nil)
       result = FocusSchema.execute(
         query,
-        variables: { input: variables }
+        variables: { input: variables },
+        context: { current_user: current_user }
```

Comment:
As far as I know, `FocusSchema.execute` didn't fetch the request header. That's why I didn't use the `execute_and_parse_graphql_response`. `FocusSchema.execute` doesn't have a request header because it's executed after the actual request.

Reply:
@VinceYoon oh I see well never mind